examples/ntptfs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "ntptfs-winfsp-rs"
-version = "0.1.0-alpha"
+name = "ntptfs"
+version = "0.0.0-alpha"
```

Comment:
Don’t change the version number or the package name please.

Reply:
Done